nucliadb/nucliadb/common/maindb/tikv.py

```diff
@@ -62,7 +63,14 @@ async def abort(self):
 
     async def commit(self):
         with tikv_observer({"type": "commit"}):
-            await self.txn.commit()
+            try:
+                await self.txn.commit()
+            except Exception as exc:
+                exc_text = str(exc)
+                if "WriteConflict" in exc_text:
+                    raise ConflictError(exc_text) from exc
+                else:
+                    raise
```

Comment:
WDYT about adding another label to the `tikv_observer` metric to track how many conflicts and other errors we get on commits?

Reply:
agreed, done.